logicmonitor_data_sdk/models/rest_metrics_v1.py

```diff
@@ -24,315 +24,328 @@
 import six
 
 from logicmonitor_data_sdk.models.list_rest_data_source_instance_v1 import \
-  ListRestDataSourceInstanceV1  # noqa: F401,E501
+    ListRestDataSourceInstanceV1  # noqa: F401,E501
 from logicmonitor_data_sdk.models.map_string_string import \
-  MapStringString  # noqa: F401,E501
+    MapStringString  # noqa: F401,E501
 
 
 class RestMetricsV1(object):
-  """NOTE: This class is auto generated by the swagger code generator program.
+    """NOTE: This class is auto generated by the swagger code generator program.
 
   Do not edit the class manually.
   """
 
-  """
+    """
```

Comment:
can u check the code formatting again?? seems that we have added some spaces in the files and because of that diff is large...